utils/PathSanitizingFileCheck

```diff
@@ -80,7 +80,7 @@ constants.""")
         # Since we want to use pattern as a regex in some platforms, we need
         # to escape it first, and then replace the escaped slash
         # literal (r'\\/') for our platform-dependent slash regex.
-        stdin = re.sub(re.sub(b'\\\\/', slashes_re, re.escape(pattern)),
+        stdin = re.sub(re.sub(br'[/\\]', slashes_re, re.escape(pattern)),
```

Comment:
I imagine the need for both `/`and `\` comes from Windows. If I understand correctly you want both in the pattern to be replaced by the `slashes_re`. This was not normally needed in Windows because if part of the pattern (normally the start) used `\`, it would have also used `\` in the results. Anyway, I think the value you want should be `br'(\\/|\\\\)'` because the `re.escape(pattern)` at the end escapes the slashes (and other crap that might trip the regex engine if used verbatim).

Reply:
hmm, interesting, I'll try that; note that I'm going to be closing this PR off and creating a new one because I found that I needed more changes :-(